src/app/clusters/groups-server/groups-server.cpp

```diff
@@ -289,7 +289,7 @@ bool emberAfGroupsClusterGetGroupMembershipCallback(app::CommandHandler * comman
         iter = provider->IterateEndpoints(fabricIndex);
         VerifyOrExit(nullptr != iter, status = EMBER_ZCL_STATUS_FAILURE);
 
-        err = commandObj->AddResponseData(commandPath, GroupMembershipResponse(commandData, commandPath.mEndpointId, iter));
+        err = commandObj->AddResponse(commandPath, GroupMembershipResponse(commandData, commandPath.mEndpointId, iter));
         iter->Release();
         status = (CHIP_NO_ERROR == err) ? EMBER_ZCL_STATUS_SUCCESS : EMBER_ZCL_STATUS_FAILURE;
```

Comment:
This is going to try to emberAfSendImmediateDefaultResponse in the exit block.  But we actually do want that for exits from earlier in this function....  Followup for making this codepath sane is fine, or you could make it sane here (e.g. by setting status to success no matter what AddResponse returned, or just logging and returning here instead of falling through to the exit block, or something.